source/documentation/values/calculations.html.md.erb

```diff
@@ -210,12 +233,34 @@ combined at build time, so `min(12px % 10, 10%)` will throw an error.
   from, or compared to numbers with units. `min()` and `max()` are different,
   though: for backwards-compatibility with the global Sass `min()` and `max()`
   functions which allow unit/unitless mixing for historical reasons, these units
-  can be mixed as long as they're contained directly within a `min()` or `max()`
+  can be mixed as long as they're contained directly within a `min()` or `max()` 
   calculation.
+  For instance, `sin($number: 4)` will throw an error, as `sin()` was 
+  never a global Sass function.
 <% end %>
```

Comment:
Since this `heads_up` is specific to `min` and `max`, it should probably be under that heading (if you end up keeping it).